README.md

```diff
@@ -28,7 +28,7 @@ regularly to indicate new official, stable release versions of Raven Core.
 
 The contribution workflow is described in [CONTRIBUTING.md](CONTRIBUTING.md).
 
-Developer IRC can be found on Freenode at #raven-core-dev.
+Developer IRC is inactive please join us on discord in #development. https://discord.gg/jn6uhur
```

Comment:
Not sure if that link is set to never expire and max number of people.  This one has those features and is set to development channel rather than the nest: https://discord.gg/fndp4NBGct.

Reply:
Fair enough, good catch. I used the link from the ravencoin website which I imagine has those features. I am new to github, do I need to submit a new pull request or is there a way to edit this one?

Reply:
You can just make new commits to the same branch and push your changes.

When merging, the maintainer will combine them into one commit and tweak to make it tidy and nice.

Reply:
Thank you, I will target the develop branch and send a new pull request. Thanks for the info!